factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/internal/listen/PgListener.java

```diff
@@ -50,55 +49,59 @@
 @RequiredArgsConstructor
 public class PgListener implements InitializingBean, DisposableBean {
 
+    private static final int MAX_ALLOWED_NOTIFICATION_LATENCY_IN_MILLIS = 200;
+
     @NonNull
     final PgConnectionSupplier pgConnectionSupplier;
 
     @NonNull
     final EventBus eventBus;
 
-    @NonNull
-    final Predicate<Connection> pgConnectionTester;
-
     private final AtomicBoolean running = new AtomicBoolean(true);
 
     private Thread listenerThread;
 
-    private final int blockingWaitTimeInMillis = 1000 * 60;
+    private final int blockingWaitTimeInMillis = 1000 * 15;
```

Comment:
how about making those values configurable?

Reply:
I assume you mean globally configurable when you run the FC?